src/artifactContentRenderers/renderingArtifactContentRenderer.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ */
+
+import * as vscode from 'vscode';
+import * as sarif from 'sarif';
+import { ArtifactContentRenderer } from './artifactContentRenderer';
+
+/**
+ * Class used to render artifact content objects with text.
+ */
+export class RenderingArtifactContentRenderer implements ArtifactContentRenderer {
```

Comment:
>RenderingArtifactContentRenderer [](start = 13, length = 32)

Name this to rendered

Reply:
Filename as well

---
In reply to: [412388852](https://github.com/microsoft/sarif-vscode-extension/pull/252#discussion_r412388852) [](ancestors = 412388852)